.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Community Support
+    url: https://github.com/orgs/community/discussions
+    about: Please ask and answer questions here.
+  - name: GitHub Security Bug Bounty
+    url: https://bounty.github.com/
+    about: Please report security vulnerabilities here.
```

Comment:
I'm not quite sure what this file is here for.  Is it a sample file that wasn't updated?

Reply:
@mi-hol as before, should this be here?

Reply:
I have removed the file as suggested.

Reply:
@EspoTek Oops, I unintentionally edited your question instead of creating a reply and there is no "undo" feature. Please apologize